### ISSUE TITLE
Add `sqlalchemy` namespace as `sa` to `flask shell`

### ISSUE THREAD

**Issue by yuxiaoy1** [pallets-eco/flask-sqlalchemy#1348]
I'd like to make a PR for adding `sqlalchemy` namespace as `sa` to `flask shell`, what do you think?

**Comment by davidism** [comment]
Yes, there's a few ideas I had with Flask-SQLAlchemy-Lite that I'd like to bring over. This is one of them. Happy to review a PR!
